chrome/content/coniks/context.js

```diff
@@ -6,56 +6,46 @@ Cu.import("chrome://otr/content/helpers.js");
 
 var OS;
 
-function ConiksContext(opts) {
-  if (!(this instanceof ConiksContext)) {
-    return new ConiksContext(opts);
-  }
-  opts = opts || {};
+class ConiksContext {
+  constructor(opts) {
+    opts = opts || {};
 
-  if (opts.account === null) {
-    return null;
-  }
-  this.account = opts.account;
-  if (opts.fingerprint !== undefined || opts.fingerprint !== null) {
-    this.fingerprint = opts.fingerprint;
+    if (opts.account === null) {
+      return null;
+    }
+    this.account = opts.account;
+    if (opts.fingerprint !== undefined || opts.fingerprint !== null) {
+      this.fingerprint = opts.fingerprint;
```

Comment:
Oops, `fingerprint` should belong to `AccountContext`, not `ConiksContext`. Sorry for the confusion.